script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -763,8 +762,7 @@ Every man to his trades, so isn't it fine to leave monsters to the monster-kille
 [sha:32e87a2d7159bc2763a2dcc7adf366fa0c763e01]{
 -- Page 850, Offset 13136.
 -- 「だから、<化|・><け|・><物|・><殺|・><し|・><が|・><化|・><け|・><物|・><に|・><な|・><っ|・><た|・><ら|・><困|・><る|・><ん|・><だ|・><よ|・>。
-// we'll be in trouble if the monster-killer becomes a monster in italics, ruby dots <化|・><け|・><物|・><殺|・><し|・><が|・><化|・><け|・><物|・><に|・><な|・><っ|・><た|・><ら|・><困|・><る|・><ん|・><だ|・><よ|・>
-"That's why we'll be in trouble if the monster-killer becomes a monster.
+"That's why, we'd be in trouble if the monster-killer became a monster herself.
```

Comment:
"monster hunter". monster-killer would be "both a monster and a killer", not a killer of monsters. Also, monster hunter sounds more natural, even if only because its the name of the game series :nekoarc: